introduction.md

```diff
@@ -1 +1,5 @@
-Hi! I'm Bruno Pisani! My favourite codeschool path is Git (the only one I almost completed until now), but I'm very excited to start the iOS path!
\ No newline at end of file
+Hi! I'm Bruno Pisani! 
+My favourite codeschool path is Git (the only one I almost completed until now), but I'm very excited to start the iOS path!
+* iOS
+* Game Programming
+* Web Development (Javascript, PHP, HTML, CSS, etc.)
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?